modules/postgres/src/test/scala/doobie/postgres/ReadErrorSuite.scala

```diff
@@ -29,18 +28,17 @@ class ReadErrorSuite extends munit.FunSuite {
   implicit val MyJavaEnumMeta: Meta[MyJavaEnum] = pgJavaEnum[MyJavaEnum]("myenum")
 
   test("pgEnumStringOpt") {
-    val r = sql"select 'invalid'".query[MyEnum].unique.transact(xa).attempt.unsafeRunSync()
-    assertEquals(r, Left(InvalidEnum[MyEnum]("invalid")))
+    sql"select 'invalid'".query[MyEnum].unique.transact(xa).attempt.assertEquals(Left(InvalidEnum[MyEnum]("invalid")))
   }
 
   test("pgEnum") {
-    val r = sql"select 'invalid' :: myenum".query[MyScalaEnum.Value].unique.transact(xa).attempt.unsafeRunSync()
-    assertEquals(r, Left(InvalidEnum[MyScalaEnum.Value]("invalid")))
+    sql"select 'invalid' :: myenum".query[MyScalaEnum.Value].unique.transact(xa).attempt.assertEquals(Left(
+      InvalidEnum[MyScalaEnum.Value]("invalid")))
   }
 
   test("pgJavaEnum") {
-    val r = sql"select 'invalid' :: myenum".query[MyJavaEnum].unique.transact(xa).attempt.unsafeRunSync()
-    assertEquals(r, Left(InvalidEnum[MyJavaEnum]("invalid")))
+    sql"select 'invalid' :: myenum".query[MyJavaEnum].unique.transact(xa).attempt.assertEquals(Left(
+      InvalidEnum[MyJavaEnum]("invalid")))
```

Comment:
Not a requirement, but I think these test expressions would be a bit more readable if executions and assertions parts were kept on separate lines like this:
```suggestion
    sql"select 'invalid' :: myenum".query[MyJavaEnum].unique.transact(xa).attempt
      .assertEquals(Left(InvalidEnum[MyJavaEnum]("invalid")))
```
This basically could be applied to all the test cases in the PR.

Although, you don't have to, just a suggestion. Feel free to ignore)

Reply:
Hi @satorg, Sounds good to me will apply this.